write-fonts/src/tables/glyf.rs

```diff
@@ -939,6 +1170,469 @@ mod tests {
         assert_eq!(points[3].y, -20);
     }
 
+    #[test]
+    #[should_panic(expected = "UnequalNumberOfElements")]
+    fn simple_glyphs_from_kurbo_unequal_number_of_elements() {
+        let mut path1 = BezPath::new();
+        path1.move_to((0., 0.));
+        path1.line_to((1., 1.));
+        path1.line_to((2., 2.));
+        path1.line_to((0., 0.));
+        path1.close_path();
+        assert_eq!(path1.elements().len(), 5);
+
+        let mut path2 = BezPath::new();
+        path2.move_to((3., 3.));
+        path2.line_to((4., 4.));
+        path2.line_to((5., 5.));
+        path2.line_to((6., 6.));
+        path2.line_to((3., 3.));
+        path2.close_path();
+        assert_eq!(path2.elements().len(), 6);
+
+        simple_glyphs_from_kurbo(&[path1, path2]).unwrap();
+    }
+
+    #[test]
+    #[should_panic(expected = "InconsistentPathElements")]
+    fn simple_glyphs_from_kurbo_inconsistent_path_elements() {
+        let mut path1 = BezPath::new();
+        path1.move_to((0., 0.));
+        path1.line_to((1., 1.));
+        path1.quad_to((2., 2.), (0., 0.));
+        path1.close_path();
+        let mut path2 = BezPath::new();
+        path2.move_to((3., 3.));
+        path2.quad_to((4., 4.), (5., 5.));
+        path2.line_to((3., 3.));
+        path2.close_path();
+
+        simple_glyphs_from_kurbo(&[path1, path2]).unwrap();
+    }
+
+    fn make_interpolatable_paths(
```

Comment:
this is long enough I can't tell what it does - what will the resulting paths look like - at a glance, perhaps a brief comment is warranted?

Reply:
I added a docstring in https://github.com/googlefonts/fontations/pull/368/commits/57d9d719e57de249504931dad8db44394ea78a3c